docs/data/apis/rpc/admin-guide/data-lake-integration.mdx

```diff
@@ -11,6 +11,8 @@ RPC version 23.0 introduces data lake integration for the `getLedgers` endpoint,
 
 The process involves setting up a ledger data lake and then configuring your RPC node to use it.
 
+> Note: The metadata of a ledger returned by `getLedgers` may vary depending on its source. When a ledger is retrieved from the RPC's local datastore, its metadata is subject to your RPC configuration. However, ledgers fetched from the data lake are typically stored with all metadata included.
```

Comment:
I think this is perfect for the purposes of just highlighting possible differences. I would just suggest using the special box syntax on our site for it, I think it's something like

```
::: info

your content here

:::
```